tools/issue-labeler/src/IssueLabeler/.gitignore

```diff
@@ -0,0 +1 @@
+./local.settings.json
```

Comment:
We can probably promote this pattern to the root ignore file and remove this one.